<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBNRMARITA" failures="1" tests="3" name="VerifyTerminatedStatus" time="11.839" errors="0" timestamp="2023-04-24T14:26:53 IST">
  <testcase classname="script.VerifyTerminatedStatus" name="validateTerminatedAdvancePage" time="2.637"/>
  <testcase classname="script.VerifyTerminatedStatus" name="validateNorecords" time="0.054"/>
  <testcase classname="script.VerifyTerminatedStatus" name="validateRejectedAdvanceDetails" time="3.372">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [null] but found [],
	expected [3468] but found []">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [null] but found [],
expected [3468] but found []
at script.VerifyTerminatedStatus.validateRejectedAdvanceDetails(VerifyTerminatedStatus.java:75)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validateRejectedAdvanceDetails -->
</testsuite> <!-- VerifyTerminatedStatus -->
